src/analytics/Properties.tsx

```diff
@@ -1180,6 +1181,7 @@ interface DappExplorerEventsProperties {
   [DappExplorerEvents.dapp_select]: DappEventProperties
   [DappExplorerEvents.dapp_bottom_sheet_open]: DappEventProperties
   [DappExplorerEvents.dapp_bottom_sheet_dismiss]: DappEventProperties
```

Comment:
This can be removed based if the existing DappExplorerEvents are sufficient.

Reply:
I will remove this part along with other parts related to this!